processing/src/main/java/org/apache/druid/query/QueryMetrics.java

```diff
@@ -32,47 +32,48 @@
  * Abstraction wrapping {@link org.apache.druid.java.util.emitter.service.ServiceMetricEvent.Builder} and allowing to
  * control what metrics are actually emitted, what dimensions do they have, etc.
  *
- *
- * Goals of QueryMetrics
- * ---------------------
- *  1. Skipping or partial filtering of particular dimensions or metrics entirely. Implementation could leave the body
- *  of the corresponding method empty, or implement random filtering like:
+ * <h4>Goals of QueryMetrics</h4>
```

Comment:
nit: we don't publish javadoc, and thus prefer a format that is easily readable in IDEs than in web browsers.